packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/legacy/template.html

```diff
@@ -0,0 +1,36 @@
+<div class="datacenter-backup-legacy">
+    <oui-message type="warning">
+        <ul class="list-unstyled">
+            <li>
+                <strong
+                    data-translate="datacenter_backup_legacy_message_1"
+                ></strong>
+            </li>
+            <li>
+                <span
+                    data-translate="dedicatedCloud_datacenter_backup_new_description2"
```

Comment:
Seems like not all translations are brought by this module (https://github.com/ovh/manager/pull/2337/files#diff-ce0d68bc50316c7fe729b4c84f9effcfR6)
Moreover, given the content I'm not sure a list is the best way to go. it looks like what is needed is just some `span`/`p`

Reply:
There is a translation that is shared with new order flow. So i am using the parents translation for that sentence. I can repeat the translation and make it standalone.